src/library/libraryview.h

```diff
@@ -14,6 +14,7 @@ class LibraryView {
     virtual ~LibraryView() {};
 
     virtual void onShow() = 0;
+    virtual bool hasFocus() = 0;
```

Comment:
Sorry, I didn't have a look at the inheritance hierarchy. Then we should 
leave it as it is.

On 05/22/2016 03:58 PM, Tim Rae wrote:
>
> In src/library/libraryview.h 
> <https://github.com/mixxxdj/mixxx/pull/953#discussion_r64150309>:
>
> > @@ -14,6 +14,7 @@ class LibraryView {
> >      virtual ~LibraryView() {};
> >
> >      virtual void onShow() = 0;
> > +    virtual bool hasFocus() = 0;
>
> I was originally trying to do what you said, but I don't really know how 
> to do that in C++ (I've only just started learning). LibraryView is an 
> interface that doesn't have knowledge of whether or not the classes that 
> implement it are QWidgets or not so we can't add it here. Where would I 
> put the default implementation?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/953/files/8491121006d713c19c76f205123f1d3b0ba792d6#r64150309>
>